modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/request-form-handler/index.js

```diff
@@ -10,8 +10,20 @@
  */
 
 const requestType = document.querySelector('[name="requestType"]');
+const requestStatus = document.querySelector('[name="requestStatus"]');
 const serviceForm = document.getElementsByClassName('.service-form');
 const grantForm = document.getElementsByClassName('.grant-form');
+const grantRequestType = document.querySelector('[name="grantRequestType"]');
+const grantAmount = document.querySelector('[name="grantAmount"]');
+const managerEmailAddress = document.querySelector(
+	'[name="managerEmailAddress"]'
+);
+const totalHoursRequested = document.querySelector(
+	'[name="totalHoursRequested"]'
+);
+const startDate = document.querySelector('[name="startDate"]');
+const endDate = document.querySelector('[name="endDate"]');
+
```

Comment:
Please, use ```fragmentElement``` instead ```document```

Reply:
As we are accessing elements outside the fragment, we need to use `document` in this case.